hail/python/hail/methods/impex.py

```diff
@@ -1828,8 +1828,8 @@ def import_vcf(path,
         If ``True``, create sites-only dataset. Don't load sample IDs or
         entries.
     call_fields : :obj:`list` of :obj:`str`
-        List of FORMAT fields to load as :py:data:`.tcall`. "GT" is loaded as
-        a call automatically.
+        List of FORMAT fields to load as :py:data:`.tcall`. "GT" and "PGT" are
```

Comment:
I just looked at the spec, and PGT isn't actually one of the declared fields. I think it would make more sense to just set the default call fields to `('PGT')` instead of making it hardcoded like this - For instance, a user with an in-spec float PGT field be unable to load that.